app/models/html_document.rb

```diff
@@ -73,4 +77,12 @@ def main_html
       element&.text.present?
     end&.to_html
   end
+
+  def dublin_core_date
```

Comment:
Note: I'm only using the DC data in `HtmlDocument` but not `ApplicationDocument` because I've found that application documents almost always have both a created and a modified timestamp, and any other metadata such as Dublin Core or Open Graph is redundant.